tsconfig.base.json

```diff
@@ -6,6 +6,7 @@
     "lib": ["ES2022", "DOM"],
     "module": "commonjs",
     "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
What did we need this for?

Reply:
For some reason something changed in the data model and it cannot import the json schemas without this anymore :cry: